src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/AbstractEngineSchemaTest.java

```diff
@@ -683,8 +683,9 @@ private static void checkEntity(final String reasonMessage, final DbEntity entit
                 // this second entry is needed because MySQL returns tables as SYSTEM_TABLEs (when they are created in default schema)
                 MapEntry.entry(name, DbEntityType.SYSTEM_TABLE)
         };
-        final String originalSchema = Deencapsulation.getField(originalEngine, "currentSchema");
-        final String otherSchema = Deencapsulation.getField(otherEngine, "currentSchema");
```

Comment:
These don't exist anymore on the newest version.
We would need to get the `Field` and then get the value ourselves....

It's just easier to use the already existing `getSechema()`

Reply:
Yeah, not sure why wasn't public in the first place. Do you know of any concern?

Reply:
I thought it was me, since I did a fix for PDB to properly use the schema some time ago,
but that `getSchema` method exists since this repository was created in 2014!

I don't see any problem being public, it's only "public" in the engine classes but not in the interface.
It has no use in the interface anyway, because we already know it beforehand, as we specify it in the config.